Dockerfile

```diff
@@ -1,20 +1,27 @@
-FROM ubuntu:14.04
-MAINTAINER Whit Morriss <whit.morriss@canonical.com>
+FROM ubuntu:16.04
+MAINTAINER Matthew Bruzek <Matthew.Bruzek@canonical.com>
 
-ADD setup.sh /setup.sh
-RUN /setup.sh
+RUN useradd -m ubuntu -s /bin/bash
 
-RUN mkdir /home/ubuntu/.juju
-RUN mkdir /home/ubuntu/trusty
-RUN mkdir /home/ubuntu/precise
+RUN mkdir -p \
+    /home/ubuntu/.local/share/juju \
+    /home/ubuntu/builds \
+    /home/ubuntu/trusty \
+    /home/ubuntu/xenial
```

Comment:
don't we need [layers, interfaces] here too?  charmbox inherits the volume definitions now (ever since https://github.com/juju-solutions/charmbox/pull/61/files), so i think we should add those dirs here.